bigtable-dataflow-parent/bigtable-hbase-beam/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+    <difference>
```

Comment:
please add a comment: "In the migration to v2, BigtableOptions along with bigable-client-core has been removed